index.css

```diff
@@ -222,3 +220,748 @@ section img {
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+.frame-sub {
+  display: grid;
+  grid-row: 6;
+  align-items: center;
+  padding: 0 24px;
+  gap: 24px;
+  width: 375px;
+  height: 2436px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.card-works {
+  width: 327px;
+  height: 386px;
+  position: relative;
+  margin-bottom: 3em;
+}
+
+.mask {
+  width: 327px;
+  height:
+    38
+    6px;
+  left: 0;
+  top: 0;
+}
+
+.img-placeholder {
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  width: 327px;
+  height: 386px;
+  margin-top: 4em;
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url(images/Img\ Place.png);
+}
+
+.div {
+  padding: 3em 2em;
+}
+
+.mask-sub {
+  width: 100%;
+  height: 100%;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  background: #262626;
+}
+
+.right-works {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 20px 50px !important;
+  gap: 12px;
+  position: absolute;
+  width: 327px;
+  height: 248px;
+  left: calc(40% - 327px / 2);
+  bottom: 62px;
+}
+
+.pro {
+  width: 295px;
+  height: 81px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  margin-top: 80px;
+  color: #fff;
+}
+
+.sub1 {
+  width: 295px;
+  height: 87px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.tags {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 12px;
+  width: 280px;
+  height: 40px;
+}
+
+.tags1 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 50px;
+  height: 20px;
+  list-style: none;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.tags1 h6 {
+  width: 32px;
+  height: 20px;
+  list-style: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+}
+
+.tags2 {
+  display: flex;
+  list-style: none;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 110px;
+  height: 20px;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.tags2 h6 {
+  width: 71px;
+  height: 20px;
+  list-style: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+}
+
+.tags3 {
+  display: flex;
+  list-style: none;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 61px;
+  height: 20px;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.tags3 h6 {
+  width: 37px;
+  height: 20px;
+  list-style: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+}
+
+.action {
+  position: relative;
+  height: 48px;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background: #ff6b00;
+}
```

Comment:
- [ ] As the last code reviewer said: Please add a hover and pressed state on your buttons to match the design template. You can use the :hover and :active pseudo-classes for this.

![image](https://user-images.githubusercontent.com/58642949/192076733-f42f3158-a9a3-4054-868f-72d518920d6e.png)